openapi.yaml

```
@@ -0,0 +1,547 @@
+                  model:
+                    type: string
+                    description: The model used for processing.
+              example:
+                content: "Hi there! How can I assist you today?"
+                processing_time: 3105.397454
+                processing_id: "8aa97481-20f9-48f4-a12d-d02eb6c1d62a"
+                processingCount: 84
+                conversation_id: "conv_1724236791746"
+                model: "aicon-v4-nano-160824"
+  
+  /api/ai/content/v4:
```

YAMLlint (1.35.1)

[error] duplication of key "/api/ai/content/v4" in mapping

(key-duplicates)